sdk/trace/src/main/java/io/opentelemetry/sdk/trace/samplers/ImmutableSamplingResult.java

```diff
@@ -22,6 +22,18 @@
   static final SamplingResult EMPTY_RECORDED_SAMPLING_RESULT =
       ImmutableSamplingResult.createWithoutAttributes(SamplingDecision.RECORD_ONLY);
 
+  static SamplingResult getEmptyRecordedAndSampledSamplingResult() {
```

Comment:
small thing, but I might just call this `recordAndSample()`. It's much easier to read and there's fewer words to parse through to get to what it's doing. I know this isn't public, but I think it would be more readable to have a shorter, less wordy method name.  Same for the other methods.

Reply:
actually...do we even need these methods at all? The things that need them can just use the fields like they are today. I think we should stick with that.

Reply:
yes, let's get rid of these methods and just let the package access the package-protected constants like it has been.

Reply:
@Rocksnake please get rid of these methods. thanks!

Reply:
@Rocksnake I think you missed this comment